src/components/ha-button-toggle-group.ts

```diff
@@ -11,26 +11,28 @@ import {
 import "./ha-icon-button";
 
 import { fireEvent } from "../common/dom/fire_event";
-import type { ToggleButton } from "../types";
+import type { ToggleViewButton } from "../types";
```

Comment:
Why was this changed to a less generic type?

Reply:
Because it should be a less generic type right? These buttons are only view buttons and the value can only be a Full Calendar view

Reply:
Why? It is for a button toggle group element. It doesn't have to toggle a view.

Reply:
I think I was drunk when I did this. I honestly don't know what I was thinking